src/platform/silabs/SiWx917/wifi/wfx_host_events.h

```diff
@@ -149,28 +148,19 @@ typedef enum
 /* Note that these are same as RSI_security */
```

Comment:
Are these in fact still the same as RSI_security?

But also, is there a reason this is not just using the spec-defined enum?

Reply:
now, they are not same as `rsi_security`, it was a comment which I plan to clean up during the cleanup activity.